salesagility/SuiteCRM
# Fixed  #1234 - 7.8 Upgrade returns Javascript:void(0) for new filter …

…functionality



The jssource did not contain the attribute that was being called hence the javascript failure.

## Description




Re-instated this attribute

## Motivation and Context


Resolves the Issue with javascript:void issue when clicking the new filter button.

## How To Test This


Please:
1. Rebuild Minified JS Files
2. Rebuild JS Grouping Files
3. If necessary Repair & Rebuild
4. Clear Browser cache (as you need to ensure you have the latest javascript loaded).
5. Navigate to a module with the new Filter functionality and Click on the Filter button.
6.  If successful a PopUp window will appear.

## Types of changes


### Final checklist
